pkg/operation/botanist/botanist.go

```diff
@@ -232,3 +232,13 @@ func (b *Botanist) RequiredExtensionsReady(ctx context.Context) error {
 
 	return nil
 }
+
+// IsLoggingEnabled return true if the logging stack for clusters is enabled.
+func (b *Botanist) IsLoggingEnabled() bool {
```

Comment:
unit test?

Reply:
This should probably rather be a helper function (https://github.com/gardener/gardener/tree/master/pkg/gardenlet/apis/config/helper)

Reply:
Addressed in 8c9ee3ad536361a54bdf56825cd1cb78a2bc8b72